src/js/oCrossword.js

```diff
@@ -702,40 +725,64 @@ OCrossword.prototype.assemble = function assemble() {
 
 				if(!isAndroid()) {
 					e.target.value = String.fromCharCode(e.keyCode);
```

Comment:
I think you are wanting to use [KeyboardEvent.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) here and not the deprecated KeyboardEvent.keyCode.

```suggestion
					e.target.value = e.key;
```

Reply:
Sadly, a quick test on android reveals... e.key="Unidentified" for an excitingly wide range of reasonable key presses and contexts, such as, you know, A-Z, or backspace when there is a character in the cell, so e.key is just as completely useless as e.keyCode when it comes to reading what key may (or may not) have pressed on the android keyboard.
Sigh.